diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b294fe..7f7fb1f27071b 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
+COMPANY_FULLNAME=Shiemi
+COMPANY_SHORTNAME=Shiemi
+PRODUCT_FULLNAME=Shiemi
+PRODUCT_SHORTNAME=Shiemi
+PRODUCT_INSTALLER_FULLNAME=Shiemi Installer
+PRODUCT_INSTALLER_SHORTNAME=Shiemi Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+MAC_BUNDLE_ID=com.shiemi.Shiemi
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
diff --git a/chrome/install_static/chromium_install_modes.h b/chrome/install_static/chromium_install_modes.h
index f29a8228fe611..3c3b14f0a0c07 100644
--- a/chrome/install_static/chromium_install_modes.h
+++ b/chrome/install_static/chromium_install_modes.h
@@ -21,10 +21,10 @@ inline constexpr wchar_t kCompanyPathName[] = L"";
 
 // The brand-specific product name to be included as a component of the install
 // and user data directory paths.
-inline constexpr wchar_t kProductPathName[] = L"Chromium";
+inline constexpr wchar_t kProductPathName[] = L"Shiemi";
 
 // The brand-specific safe browsing client name.
-inline constexpr char kSafeBrowsingName[] = "chromium";
+inline constexpr char kSafeBrowsingName[] = "shiemi";
 
 // Note: This list of indices must be kept in sync with the brand-specific
 // resource strings in chrome/installer/util/prebuild/create_string_rc.
@@ -45,39 +45,39 @@ inline constexpr auto kInstallModes = std::to_array<InstallConstants>({
         .logo_suffix = L"",  // No logo suffix for the primary install mode.
         .app_guid =
             L"",  // Empty app_guid since no integration with Google Update.
-        .base_app_name = L"Chromium",              // A distinct base_app_name.
-        .base_app_id = L"Chromium",                // A distinct base_app_id.
-        .browser_prog_id_prefix = L"ChromiumHTM",  // Browser ProgID prefix.
+        .base_app_name = L"Shiemi",              // A distinct base_app_name.
+        .base_app_id = L"Shiemi",                // A distinct base_app_id.
+        .browser_prog_id_prefix = L"ShiemiHTM",  // Browser ProgID prefix.
         .browser_prog_id_description =
-            L"Chromium HTML Document",  // Browser ProgID description.
-        .direct_launch_url_scheme = "chromium",
-        .pdf_prog_id_prefix = L"ChromiumPDF",  // PDF ProgID prefix.
+            L"Shiemi HTML Document",  // Browser ProgID description.
+        .direct_launch_url_scheme = "shiemi",
+        .pdf_prog_id_prefix = L"ShiemiPDF",  // PDF ProgID prefix.
         .pdf_prog_id_description =
-            L"Chromium PDF Document",  // PDF ProgID description.
+            L"Shiemi PDF Document",  // PDF ProgID description.
         .active_setup_guid =
-            L"{7D2B3E1D-D096-4594-9D8F-A6667F12E0AC}",  // Active Setup
+            L"{69AC06F5-A727-4688-BD7C-151B24D4FF52}",  // Active Setup
                                                         // GUID.
-        .toast_activator_clsid = {0x635EFA6F,
-                                  0x08D6,
-                                  0x4EC9,
-                                  {0xBD, 0x14, 0x8A, 0x0F, 0xDE, 0x97, 0x51,
-                                   0x59}},  // Toast Activator CLSID.
-        .elevator_clsid = {0xD133B120,
-                           0x6DB4,
-                           0x4D6B,
-                           {0x8B, 0xFE, 0x83, 0xBF, 0x8C, 0xA1, 0xB1,
-                            0xB0}},  // Elevator CLSID.
+        .toast_activator_clsid = {0x620E7A31,
+                                  0xF1A1,
+                                  0x4BDD,
+                                  {0xB4, 0x43, 0x05, 0x56, 0xEE, 0x61, 0x3C,
+                                   0x55}},  // Toast Activator CLSID.
+        .elevator_clsid = {0xCF3DF096,
+                           0x53F5,
+                           0x4838,
+                           {0xB7, 0x68, 0x64, 0x15, 0xD0, 0xAB, 0xE2,
+                            0x0E}},  // Elevator CLSID.
         .elevator_iid = {0xbb19a0e5,
                          0xc6,
                          0x4966,
                          {0x94, 0xb2, 0x5a, 0xfe, 0xc6, 0xfe, 0xd9,
                           0x3a}},  // IElevator IID and TypeLib
         // {BB19A0E5-00C6-4966-94B2-5AFEC6FED93A}.
-        .tracing_service_clsid = {0x83f69367,
-                                  0x442d,
-                                  0x447f,
-                                  {0x8b, 0xcc, 0x0e, 0x3f, 0x97, 0xbe, 0x9c,
-                                   0xf2}},  // SystemTraceSession CLSID.
+        .tracing_service_clsid = {0x4661a036,
+                                  0x9ecb,
+                                  0x4e4c,
+                                  {0x88, 0x19, 0xa0, 0x24, 0x60, 0x17, 0x3d,
+                                   0x69}},  // SystemTraceSession CLSID.
         .tracing_service_iid = {0xa3fd580a,
                                 0xffd4,
                                 0x4075,
diff --git a/chrome/installer/setup/generate_visual_elements_manifest_work_item.cc b/chrome/installer/setup/generate_visual_elements_manifest_work_item.cc
index 1683b08552ba3..de954ddbe3534 100644
--- a/chrome/installer/setup/generate_visual_elements_manifest_work_item.cc
+++ b/chrome/installer/setup/generate_visual_elements_manifest_work_item.cc
@@ -31,7 +31,7 @@ std::string GenerateVisualElementsManifest(const base::Version& version) {
       "      Square70x70Logo='%s\\SmallLogo%s.png'\r\n"
       "      Square44x44Logo='%s\\SmallLogo%s.png'\r\n"
       "      ForegroundText='light'\r\n"
-      "      BackgroundColor='#5F6368'/>\r\n"
+      "      BackgroundColor='#d2703f'/>\r\n"
       "</Application>\r\n";
 
   // Construct the relative path to the versioned VisualElements directory.
